src/System.Diagnostics.Process/tests/ProcessTests.netstandard1.7.cs

```diff
@@ -25,6 +25,40 @@ public void TestHandleCount()
             }
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.OSX)]
+        public void TestHandleCount_OSX()
+        {
+            using (Process p = Process.GetCurrentProcess())
+            {
+                Assert.Equal(0, p.HandleCount);
+            }
+        }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Linux | TestPlatforms.Windows)]
+        public void HandleCountChanges()
+        {
+            RemoteInvoke(() =>
+            {
+                Process p = Process.GetCurrentProcess();
+                int handleCount = p.HandleCount;
+                using (var fs1 = File.Open(Path.Combine(Path.GetTempPath(), Path.GetTempFileName()), FileMode.OpenOrCreate))
```

Comment:
Path.GetTempFileName creates a file. Where is it getting deleted? Would FileCleanupTestBase be helpful?